website/content/en/docs/olm-integration/generating-a-csv.md

```diff
@@ -1,208 +1,248 @@
+- `spec.provider` _(user)_ : the Operator provider, with a `name`; usually an organization.
+- `spec.labels` _(user)_ : a list of `key:value` pairs to be used by Operator internals.
+- `metadata.annotations.alm-examples`: CR examples, in JSON string literal format, for your CRD's. Ideally one per CRD.
+- `metadata.annotations.capabilities`: level of Operator capability. See the [Operator maturity model][olm-capabilities]
+for a list of valid values.
+- `spec.replaces`: the name of the CSV being replaced by this CSV.
+- `spec.links` _(user)_ : a list of URL's to websites, documentation, etc. pertaining to the Operator or application
+being managed, each with a `name` and `url`.
+- `spec.selector` _(user)_ : selectors by which the Operator can pair resources in a cluster.
+- `spec.icon` _(user)_ : a base64-encoded icon unique to the Operator, set in a `base64data` field with a `mediatype`.
+- `spec.maturity`: the Operator's maturity, ex. `alpha`.
 
-* `metadata.annotations.alm-examples`: CR examples, in JSON string literal format, for your CRD's. Ideally one per CRD.
-* `metadata.annotations.capabilities`: level of Operator capability. See the [Operator maturity model][olm-capabilities] for a list of valid values.
-* `spec.replaces`: the name of the CSV being replaced by this CSV.
-* `spec.links` _(user)_ : a list of URL's to websites, documentation, etc. pertaining to the Operator or application being managed, each with a `name` and `url`.
-* `spec.selector` _(user)_ : selectors by which the Operator can pair resources in a cluster.
-* `spec.icon` _(user)_ : a base64-encoded icon unique to the Operator, set in a `base64data` field with a `mediatype`.
-* `spec.maturity`: the Operator's maturity, ex. `alpha`.
-
-## Further Reading
-
-* [Information][doc-csv] on what goes in your CSV and CSV semantics.
-* The original [design doc][doc-csv-design], which contains a thorough description how CSV's are generated by the SDK.
 
-[doc-csv]:https://github.com/operator-framework/operator-lifecycle-manager/blob/4197455/Documentation/design/building-your-csv.md
 [olm]:https://github.com/operator-framework/operator-lifecycle-manager
-[generate-csv-cli]:/docs/cli/operator-sdk_generate_csv
-[doc-csv-design]:https://github.com/operator-framework/operator-sdk/blob/master/design/milestone-0.2.0/csv-generation.md
-[doc-bundle]:https://github.com/operator-framework/operator-registry/blob/6893d19/README.md#manifest-format
+[doc-csv]:https://github.com/operator-framework/operator-lifecycle-manager/blob/0.15.1/doc/design/building-your-csv.md
+[cli-gen-kustomize-manifests]:/docs/new-cli/operator-sdk_generate_kustomize_manifests
+[cli-gen-bundle]:/docs/new-cli/operator-sdk_generate_bundle
+[cli-gen-packagemanifests]:/docs/new-cli/operator-sdk_generate_packagemanifests
+[bundle]:https://github.com/operator-framework/operator-registry/tree/v1.12.4#manifest-format
```

Comment:
update this to 'https://github.com/operator-framework/operator-registry/blob/v1.12.6/docs/design/operator-bundle.md'?
reasons: we have go.mod pointing to 1.12.6 version of operator-registry; we want readers to understand the the bundle design rather than just the high-level overview.

Reply:
I'll change this link to the doc you suggested.

This tag could be anything after `v1.6.x`, it's pinned arbitrarily to a version I know contains valid information.